projects/archiver/downloader.ts

```diff
@@ -9,12 +9,20 @@ import {
     Front,
     Image,
 } from './common'
-import { attempt, Attempt, hasSucceeded } from '../backend/utils/try'
+import {
+    attempt,
+    Attempt,
+    hasSucceeded,
+    hasFailed,
+    withFailureMessage,
+} from '../backend/utils/try'
+import { PassThrough, Readable } from 'stream'
+import { ImageSize } from '../common/src'
 
-const URL =
-    process.env.backend !== undefined
-        ? `https://${process.env.backend}`
-        : 'http://localhost:3131/'
+export const URL = 'https://d2cf1ljtg904cv.cloudfront.net/'
```

Comment:
Do we want this changed back? Was this just for debugging?

Reply:
Yes. Yes. 

Ooops.